autogen.sh

```diff
@@ -15,13 +15,26 @@ cd "$srcdir"
 
 PKG_NAME=$(autoconf --trace 'AC_INIT:$1' configure.ac)
 
-gtkdocize --copy || exit 1
+enable_gtk_doc=--enable-gtk-doc
+gtkdocize || {
```

Comment:
Also this won't distinguish the scenario when the command is not available and actual failure from running it. Please add a separate check for the command availability.